fonts/Warframe-symbols.css

```
@@ -1,912 +1,929 @@
 @font-face {
-  font-display: block;
-  font-family: "Warframe-symbols";
-  src: url("./Warframe-symbols.eot?912bd11150a8014816fb4d20280beccc#iefix") format("embedded-opentype"),
-url("./Warframe-symbols.woff?912bd11150a8014816fb4d20280beccc") format("woff"),
-url("./Warframe-symbols.woff2?912bd11150a8014816fb4d20280beccc") format("woff2"),
-url("./Warframe-symbols.ttf?912bd11150a8014816fb4d20280beccc") format("truetype"),
-url("./Warframe-symbols.svg?912bd11150a8014816fb4d20280beccc#Warframe-symbols") format("svg");
+    font-family: "Warframe-symbols";
```

Stylelint (17.12.0)

[error] Expected no quotes around "Warframe-symbols" (font-family-name-quotes)

(font-family-name-quotes)